zip-0000.rst

```diff
@@ -623,27 +625,46 @@ Consensus ZIP, a Deployment section MUST be present in order for the ZIP
+
+A ZIP SHOULD NOT be changed from a non-Released status to a Released
+status if there is significant community opposition to its content.
+(However, Draft ZIPs explicitly MAY describe proposals to which there
+is, or could be expected, significant community opposition.)
+
+A Released ZIP MUST NOT be changed to a non-Released status if the
+specification is already implemented and is in common use, or where a
+Process ZIP still reflects a consensus of the community.
+
+A Standards ZIP SHOULD only change status from Proposed to Implemented
+once the Owners provide an associated reference implementation. For
+Consensus ZIPs, an implementation MUST have been merged into at least
+one consensus node codebase (currently zcashd and/or zebra), typically
+in the period after the network upgrade's specification freeze but before
 the implementation audit. If the Owners miss this deadline, the Editors
 or Owners MAY choose to update the Deployment section of the ZIP to
 target another upgrade, at their discretion.
 
-A Process or Informational ZIP SHOULD change status from Draft to Active
-when it achieves rough consensus on the forum or PR. Such a proposal is
-said to have rough consensus if it has been open to discussion on the
-forum or GitHub PR for at least one month, and no person maintains
-any unaddressed substantiated objections to it. Addressed or obstructive
-objections can be ignored/overruled by general agreement that they have
-been sufficiently addressed, but clear reasoning MUST be given in such
-circumstances.
-
-When an Active or Final ZIP is no longer relevant, its status SHOULD be
-changed to Obsolete. This change MUST also be objectively verifiable
-and/or discussed. Final ZIPs MAY be updated; the specification is still
-in force but modified by another specified ZIP or ZIPs (check the
-optional Updated-By header).
+A Process or Informational ZIP SHOULD change status from Proposed to
+Active when it achieves rough consensus on the forum or PR. Such a
+proposal is said to have rough consensus if it has been complete and
+open to discussion on the forum or GitHub PR for at least one month,
+has been in Proposed status for at least one week, and no person
+maintains any unaddressed substantiated objections to it. Addressed
+or obstructive objections can be ignored/overruled by general agreement
+that they have been sufficiently addressed, but clear reasoning MUST be
+given in such circumstances.
+
+When an Active, Implemented, or Final ZIP is no longer relevant –for
+example because its implementation has fallen out of use or its process
+is no longer followed– its status SHOULD be changed to Obsolete. This
+change MUST also be objectively verifiable and/or discussed. Final ZIPs
+MAY be updated; the specification is still in force but modified by
+another specified ZIP or ZIPs (check the optional Updated-By header).
+
+If a non-editorial update is made to an Obsolete or Withdrawn ZIP, its
```

Comment:
This should say Obsolete, Withdrawn, or Rejected. (If it was Rejected then it should be moved to Draft.)